packages/device_info/android/src/main/java/io/flutter/plugins/deviceinfo/DeviceInfoPlugin.java

```diff
@@ -60,6 +66,7 @@ public void onMethodCall(MethodCall call, Result result) {
       build.put("tags", Build.TAGS);
       build.put("type", Build.TYPE);
       build.put("isPhysicalDevice", !isEmulator());
+      build.put("deviceId", getDeviceId());
```

Comment:
This ID doesn't really identify the device, as you write below. Maybe rename it to `androidId`?